packages/test/test-end-to-end-tests/src/test/gc/gcSweepDataStores.spec.ts

```diff
@@ -211,8 +211,11 @@ const summarizationWithUnreferencedDataStoreAfterTime = async () => {
 	const summaryVersion = (await ensureSynchronizedAndSummarize(summarizer1)).summaryVersion;
 
 	// Close the summarizer so that it doesn't interfere with the new one.
+	container.close();
 	summarizingContainer1.close();
 
+	clock.tick(sweepTimeoutMs + 10);
```

Comment:
Why'd you decide to move this?

Reply:
If it's done after loading the new summarize, its session will expire when the clock moves past sweep timeout.